python/lsst/ts/standardscripts/data/scripts/maintel/m1m3/check_actuators.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
+# This file is part of ts_standardscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
```

Comment:
Missing comment line: "# along with this program. If not, see <https://www.gnu.org/licenses/>."

Reply:
Now that you comment on this, I see that lots of scripts on this repo have that line missing. However, I see that line on the `check_actuators.py`. Let me know if you still don't see it.

Reply:
It's in the check_actuators.py but not in the data/script check_actuators.py i believe. For the other files, we are currently updating it on another PR

Reply:
https://github.com/lsst-ts/ts_standardscripts/pull/118